test/imagenator/utilities.py

```diff
@@ -0,0 +1,70 @@
+import filecmp
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from harmony import Client
+
+def print_success(msg: str) -> None:
+    print(f'\033[92mSuccess: {msg}\033[0m')
+
+def print_error(msg: str) -> None:
+    print(f'\033[91mError: {msg}\033[0m')
```

Comment:
These are duplicates of common functions in `shared_utils`. Given the time crunch for `git lfs` stuff tomorrow, we don't need to look at this too much right now, but it would be good to reuse those functions. Also, we now have at least 3 test suites assessing the output from chains ending with HyBIG. There's probably some shared functions we can add that will do that at a future date.